tests/test_metric_tz.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+import unittest
+
+from .common import BaseTest
+
+import datetime
+from c7n.output import MetricsOutput
+import os
+
+class metrics_log_test(BaseTest):
+
+  def test_output_tz(self):
+
+    mo = MetricsOutput(None)
+    c7nmetric = os.getenv("C7N_METRICS_TZ")
+
```

Comment:
generally stylistically we don't blank new line for each line of code, its more about logical grouping in terms of when we blank newline for readability.